source/extensions/quic/connection_id_generator/quic_lb/compile_bpf.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Usage: ./compile_bpf.sh <path to bpf_asm>
+# bpf_asm can be found at https://github.com/torvalds/linux/blob/master/tools/bpf/bpf_asm.c
```

Comment:
Hm. If I'm understanding this correctly, this .sh script is run manually to generate the opcodes in `source/extensions/quic/connection_id_generator/quic_lb/quic_lb.cc`. Is that right? And running this script requires having downloaded and compiled bpf_asm?

If so, can we make this a bit more automated? Can we put pbf_asm.c either in the tree or pull it in via DEPS? If so, we can generate the opcodes as part of Bazal and then #include it into quic_lb.cc which would be a bit more bullet proof. WDYT?